list.js

```diff
@@ -262,7 +262,7 @@ function getInfoFromS3Data(xml) {
     // clang-format on
   });
   if ($(xml.find('IsTruncated')[0]).text() == 'true') {
-    var nextMarker = $(xml.find('NextMarker')[0]).text();
+    var nextMarker = xml.find('NextMarker').textContent
```

Comment:
```suggestion
    var nextMarker = xml.find('NextMarker').text()
```

Also you might want to drop the `encodePath()` L225. The next marker value is not correctly retrieved so the folders are truncated (https://github.com/NixOS/infra/pull/923).

Reply:
Or more ideally, drop the call to `encodeURIComponent()` in `getInfoFromS3Data()` and adjust the condition `info.nextMarker != "null"` to `info.nextMarker !== null`